Kickstarter-iOS/Features/PledgedProjectsOverview/CardView/PPOProjectCard.swift

```diff
@@ -5,15 +5,20 @@ import SwiftUI
 
 struct PPOProjectCard: View {
   @StateObject var viewModel: PPOProjectCardViewModel
+  var parentSize: CGSize
+
+  var onShowProject: ((PPOProjectCardModel) -> Void)? = nil
+  var onSendMessage: ((PPOProjectCardModel) -> Void)? = nil
+  var onPerformAction: ((PPOProjectCardModel, PPOProjectCardModel.Action) -> Void)? = nil
```

Comment:
What do you think of getting rid of `onShowProject` and `onSendMessage` and just handling all actions in `onPerformAction`? I don't think sending a message needs to be a unique thing. And show project isn't actually an action until v2.

Reply:
The `PPOProjectCardModel.Action` type is specifically related to the buttons (as that is parsed from the GQL query). If we want to centralize around a single action and enum, then I would use a different enum to convey that from the Action.

That's a bigger question, so I'm going to merge this as-is, and 'll make a new PR for this change for future discussion.

Reply:
New PR raised in #2190